sandbox/Layout.qml

```diff
@@ -0,0 +1,276 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Layout 0.1
+
+GridLayout {
+    columns: 6
+    columnSpacing: 5
+    rowSpacing: 5
+    property ThemePalette theme
+
+    Button {
+        id: btn
+        text: "Append"
+        onClicked: {
+            buttons.append({
+                name: "Test community",
+                tooltipText: "Test Community"
+            })
+        }
+    }
+
+    StatusAppNavBar {
+        navBarChatButton: StatusNavBarTabButton {
+            icon.name: "chat"
+            badge.value: 33
+            badge.visible: true
+            tooltip.text: "Chat"
+        }
+    }
+
+    StatusAppNavBar {
+        onAdded: function(h) {
+            btn.text = h
+        }
```

Comment:
I'm using this handler to debug the problem I'm running into: Mainly, updating the `height` of the `ScrollView` used inside `StatusAppNavBar`.

This handler here proves that the `height` emitted by the handler is always `0`